src/circuits/tests.rs

```diff
@@ -258,9 +261,9 @@ mod test {
     // Passing an invalid leaf balance as input for the witness generation should fail the permutation check between the computed root hash and the instance column root hash
```

Comment:
I actually just realized that my previous test was designed wrongly as I was invalidating the leaf_hash rather than the leaf_balance. 

Now it makes more sense as there are 2 permutation checks that are failing:
- for the balance of the user for the first asset that doesn't match the expected one at row 1
- for the root hash that doesn't match the expected one passed at row 3 

Can you replace it to the comment of the test.

Also it would be cool to add another test where we invalidate the user_balance for the second asset. In that case I expect it to fail the on row 2 rather than row 1.